### PR TITLE
feat: Autodeployment scripts and containers

### PR DESCRIPTION
**Pull request by VonLatvala**
To run on container host, commands docker-compose to pull and deploy, using webhook

### CODE REVIEW

File: docker/deployment-webhook.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM library/debian:bullseye
+ENV DOCKERVERSION=18.03.1-ce
```
Comment (Tapiiri) [inline docker/deployment-webhook.Dockerfile]:
```suggestion
ENV DOCKER_VERSION=18.03.1-ce
```

File: autodeploy/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+cd
+git clone https://github.com/TiK15Ryyb/Approbatur.app.git
```
Comment (Tapiiri) [inline autodeploy/docker-entrypoint.sh:3]: If the repo would be private, here we would get through it using a Github access token, for example (now this is enough)

**Review by Tapiiri — CHANGES_REQUESTED** [review]

File: docker/deployment-webhook.Dockerfile (hunk 1 above)
Comment (VonLatvala) [reply docker/deployment-webhook.Dockerfile]: `:%s/DOCKERVERSION/DOCKER_VERSION/g`

File: autodeploy/docker-entrypoint.sh (hunk 2 above)
Comment (VonLatvala) [reply autodeploy/docker-entrypoint.sh:3]: Indeed, good point. Let's ride on the waves of public repos here ;)

**Review by Tapiiri — APPROVED** [review]
Great job! 💯

**Pull request merged**